opm/material/fluidsystems/blackoilpvt/GasPvtMultiplexer.hpp

```diff
@@ -116,6 +116,10 @@ class GasPvtMultiplexer
      */
     void initFromDeck(DeckConstPtr deck, EclipseStateConstPtr eclState)
     {
+        bool enableGas = deck->hasKeyword("GAS");
```

Comment:
IMO they should not, because these consts only make the code more ugly (IMO) and less flexible (I often had the case where I later wanted to modify them in the function) while not changing anything from the POV of the generated binary. I agree that references are a different matter, though.